main.py

```diff
@@ -0,0 +1,47 @@
+""" This file is main file of this project, here we are using most of the method and class.
+Here the date is imported, then the object are created and their parameters
+are calculated and exported after statistical transformations"""
+import json
+
+import bearing
+import shaft
+import reading_data
+from add_to_dict import adding_to_dict
+
+PATH_CSV = "dataset.csv"
+PATH_JSON = "data.json"
+
+if __name__ == "__main__":
+    values_from_csv = reading_data.reading_csv(PATH_CSV)
+    values_from_json = reading_data.reading_json(PATH_JSON)
+    items = []
+    for items_group in values_from_json.values():
+        for item in items_group:
+            if 's' in item['id']:
```

Comment:
I see that you try to recognize whether this is a shaft or bearing just by it's name. What if I call the bearing "lower_shaft_bearing"?
That's why from the beginning you gad a json with bearings and shafts separated under different keys, you could take advantage of that and just parse the bearings and shafts in separate loops for clarity.